src/Shared.CLI/Helpers/PackageDownloader.cs

```diff
@@ -22,10 +22,10 @@ public class PackageDownloader
         /// Constructor - creates a class object for downloading packages.
         /// </summary>
         /// <param name="purl">The package to download.</param>
-        /// <param name="httpClientFactory">The <see cref="IHttpClientFactory"/> to use to make the client to use when downloading.</param>
+        /// <param name="projectManagerFactory">The <see cref="ProjectManagerFactory"/> to use to get the project managers.</param>
         /// <param name="destinationDir">The directory where the package needs to be downloaded to.</param>
         /// <param name="doCaching">Check and use the cache if it exists - create if not.</param>
-        public PackageDownloader(PackageURL purl, IHttpClientFactory? httpClientFactory, string? destinationDir = null, bool doCaching = false)
+        public PackageDownloader(PackageURL purl, ProjectManagerFactory projectManagerFactory, string? destinationDir = null, bool doCaching = false)
```

Comment:
Its inconsistent that this requires a ProjectManagerFactory isntead of being able to intantiate a default one.